box-browse-sdk/src/main/java/com/box/androidsdk/browse/fragments/BoxBrowseFragment.java

```diff
@@ -183,6 +180,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         // This is a work around to show the loading circle because SwipeRefreshLayout.onMeasure must be called before setRefreshing to show the animation
         mSwipeRefresh.setProgressViewOffset(false, 0, (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, 24, getResources().getDisplayMetrics()));
 
+        // Hide the recent search view. This is specific to search fragment.
```

Comment:
See comment in the layout. Basically I think it would be better for the search fragment to load a different layout than have the parent know about this.